src/api/controllers/articleController.js

```diff
@@ -35,6 +36,7 @@ export default class ArticleController {
 
     const articleValidInput = await articleValidator(req.body);
     const { category: id } = req.body;
+    const readTime = generateReadtime(req.body.body);
```

Comment:
what is req.body.body??

Reply:
you can use articleValidInput to get the body

Reply:
I just want to know what will be the result of that function if the body is empty

Reply:
`req.body` is the body of the express req object. 
Now, the body of the article is also indexed as `body`
Reason why I have `req.body.bod`